engine/access/rest/websockets/data_providers/transaction_statuses_provider_test.go

```diff
@@ -282,6 +284,9 @@ func (s *TransactionStatusesProviderSuite) TestMessageIndexTransactionStatusesPr
 		responses = append(responses, txStatusesRes)
 	}
 
+	// Wait for the provider goroutine to finish
+	<-done
```

Comment:
Lets use https://github.com/onflow/flow-go/blob/master/utils/unittest/unittest.go#L213. Otherwise good to go